.github/pull_request_template.md

```diff
@@ -6,6 +6,9 @@
 
 (Details)
 
+We require adding a link to the [workflow](https://github.com/mlflow/mlflow/actions/workflows/pipeline-template.yml) for testing the [mlflow/mlp-regression-template](https://github.com/mlflow/mlp-regression-template) code path.
```

Comment:
I'm not sure we should keep the pull request template here either. The goal of the mlp-regression-template repo is for organizations to clone the repo and make commits for tuning / improving their models and MLOps workflows. In contrast, this PR template is more focused on the "contributing updates to MLflow Pipelines" use case.

Reply:
cc @jinzhang21

Reply:
I think it is fine to leave the PR template here. In medium term, we need to start releasing MLP template with major and minor version in sync with MLflow release, so that users can get consistent (but not necessarily backward compatible) behavior when this template is used in conjunction with MLflow Pipelines. By then, I can easily see users start contributing to this template itself (e.g. fixing bugs, adding new unit tests etc).

I'm more concerned with how to operationalize this requirement. Attaching a PR filed against MLflow repo?